synapse/tests/test_model_inet.py

```diff
@@ -776,6 +780,11 @@ def test_http_header_name(self):
             expected = (('3300:100::', '3300:100:1::ffff'), {'subs': {'min': '3300:100::', 'max': '3300:100:1::ffff'}})
             self.eq(t.norm(valu), expected)
 
+            valu = '2001:db8::/101'
+
+            expected = (('3300:100::', '2001:db8:0000:0000:0000:0000:07ff:ffff'),
+                        {'subs': {'min': '3300:100::', 'max': '2001:db8:0000:0000:0000:0000:07ff:ffff'}})
```

Comment:
Add `self.eq(t.norm(valu), expected)` so that the IPv6Range._normPyStr func is covered

Reply:
yep